pkg/nsx/services/builder.go

```diff
@@ -1033,7 +1033,8 @@ func (service *SecurityPolicyService) matchExpressionOpInExist(
 // Currently NSX only supports 'EQUALS' not 'In'. So, we have to make each value to be AND with other expressions
 // and finally produce a union set to translate from K8s 'In' to NSX EQUALS'.
 // e.g. - {key: k1, operator: NotIn, values: [a1,a2]}
-//      - {key: k2, operator: In, values: [a3,a4]}
+//   - {key: k2, operator: In, values: [a3,a4]}
+//
```

Comment:
The reason of introducing this change?

Reply:
<img width="1010" alt="image" src="https://user-images.githubusercontent.com/4385934/227832667-369849aa-3e99-4c6a-bbb8-74e9a2b05b07.png">
I see it is from 2022.6.10, @timdengyun , could you confirm it? Refactored a long time ago.

Reply:
what changes need to be confirmed? you mean @zhengxiexie @heypnus  mergeSelectorMachExpresssion func?

Reply:
//   - {key: k2, operator: In, values: [a3,a4]}
@timdengyun should this line leaved or removed?

Reply:
We noted in main branch: https://github.com/vmware-tanzu/nsx-operator/blob/eb5ba400e574c2e3553df1e4310758f1ee185a83/pkg/nsx/services/securitypolicy/builder.go#L1012
we still have - {key: k2, operator: In, values: [a3,a4]}, not sure why it was removed in the builder.go.

We need to keep this line in the comments section. Since the example here is to explain how two expressions
on  operator NotIn and operator IN  are needed to be translated into  two equivalent expressions:
{k1 NotIn [a1,a2]} AND {k2 EQUALS a3} OR {k1 NotIn [a1,a2]} AND {k2 EQUALS a4}

// e.g. - {key: k1, operator: NotIn, values: [a1,a2]}
//   - {key: k2, operator: In, values: [a3,a4]}
// The above two expressions will be translated to:
// => {k1 NotIn [a1,a2]} AND {k2 EQUALS a3} OR {k1 NotIn [a1,a2]} AND {k2 EQUALS a4}